src/Components/Card.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import './Card.css';
+import { Data } from "./Data";
+
+const Style_of_q = {
+    fontWeight: 'bold'
+};
+
+const Card = () => {
+    const [clicked, setClicked] = useState(false)
```

Comment:
Check out this review
https://github.com/meltaway/dsc-studygroup-react/pull/1#discussion_r606843506